src/util/redirects-raw.txt

```diff
@@ -0,0 +1,34 @@
+/docs/configurations/overview/                   /docs/configure/overview/                                                      301
+/docs/configurations/options-parameter/          /docs/configure/features-and-behavior/                                         301
+/docs/configurations/default-config/             /docs/configure/overview/                                                      301
+/docs/configurations/custom-webpack-config/      /docs/configure/webpack/                                                       301
+/docs/configurations/custom-babel-config/        /docs/configure/babel/                                                         301
+/docs/configurations/typescript-config/          /docs/configure/typescript/                                                    301
+/docs/configurations/add-custom-head-tags/       /docs/configure/story-rendering#adding-to-head                                 301
+/docs/configurations/add-custom-body/            /docs/configure/story-rendering#adding-to-body                                 301
+/docs/configurations/serving-static-files/       /docs/configure/images-and-assets#serving-static-files-via-storybook/          301
+/docs/configurations/env-vars/                   /docs/configure/environment-variables/                                         301
+/docs/configurations/theming/                    /docs/configure/theming/                                                       301
+/docs/configurations/cli-options/                /docs/api/cli-options/                                                         301
+/docs/formats/component-story-format/            /docs/api/csf/                                                                 301
+/docs/formats/mdx-syntax/                        /docs/api/mdx/                                                                 301
+/docs/addons/introduction/                       /docs/addons/introduction/                                                     301
+/docs/addons/using-addons/                       /docs/addons/install-addons/                                                   301
+/docs/addons/writing-addons/                     /docs/addons/writing-addons/                                                   301
+/docs/addons/api/                                /docs/addons/addons-api/                                                       301
+/docs/presets/introduction/                      /docs/api/presets/                                                             301
+/docs/presets/writing-presets/                   /docs/api/writing-presets/                                                     301
+/docs/testing/manual-testing/                    /docs/writing-tests/introduction/                                              301
+/docs/workflows/testing-with-storybook/          /docs/writing-tests/introduction/                                              301
```

Comment:
@domyen — So, looking at the [6.0 TOC](https://github.com/storybookjs/storybook/blob/release/6.0/docs/toc.js)[^1], it would seem to me that everything above this (`/docs/workflows/...`) would be "pre-6.0" and thus _shouldn't_ have the framework in the "from" path (and should just hard-code `react` in the "to" path?).

[^1]: For example, `/docs/configurations/*` and `/docs/formats/*` paths don't exist in that file, and the `/docs/addons/*` and `/docs/presets/*` paths seem to follow a slightly different structure.

Reply:
Ahh, this is further confirmed by https://github.com/storybookjs/storybook/tree/release/5.3/docs/src/pages, which shows page directories for "configurations", "formats", "testing", "addons", and "presets".

I just pushed a commit to reflect this.